packages/web-twig/src/Resources/components/Header/stories/HeaderInvertedWithActionsAndDropdown.twig

```diff
@@ -0,0 +1,92 @@
+<Header color="inverted">
+    {% include '@components/Header/stories/_partials/SpiritLogo.twig' %}
+
+    <HeaderMobileActions dialogId="header_dialog_example_3_main_navigation">
```

Comment:
And isn't this `mobile_navigation` or `user_mobile_navigation`?

Reply:
Actually, `dialogId` has no place here, `HeaderMobileActions` has no prop like that. But thanks for pointing it out! 🙂